iterator.js

```diff
@@ -14,11 +14,15 @@ function Iterator (db, options) {
 
 util.inherits(Iterator, AbstractIterator)
 
-Iterator.prototype.seek = function (key) {
-  if (typeof key !== 'string')
-    throw new Error('seek requires a string key')
+Iterator.prototype.seek = function (target) {
```

Comment:
what if target is neither a string nor a buffer here? this would be more in line with previous behaviour if it would throw in those cases